prime-angular-frontend/src/app/features/enrolment/shared/services/enrolment-state.service.ts

```diff
@@ -192,7 +192,7 @@ export class EnrolmentStateService {
       hasCertification: [null, [FormControlValidators.requiredBoolean]],
       certifications: this.fb.array([]),
       isDeviceProvider: [null, [FormControlValidators.requiredBoolean]],
-      deviceProviderNumber: ['', []],
+      deviceProviderNumber: ['', [Validators.pattern('[0-9]+'), Validators.minLength(5), Validators.maxLength(5)]],
```

Comment:
If we can make our own validators, we should probably make an exactLenth validator of some kind. I could see this being used again.

Reply:
Yes, this was just my first cut at this - I would think Martin can do a better job of this.